source/guide/3.2/quickstart-openstack.md

```diff
@@ -1,31 +1,54 @@
 ---
```

Comment:
is this some github issue? according to my understanding this file shouldnt exist anymore correct?
quickstart-openstack was renamed quickstart-real-cloud?